.github/workflows/main.yml

```diff
@@ -0,0 +1,38 @@
+name: 'Build and publish docker images (all platforms)'
+
+on:
+    push:
+        branches:
+            - master
+
+jobs:
+    release:
+        runs-on: ubuntu-latest
+        steps:
+        -
+            name: Checkout
+            uses: actions/checkout@v2
+        -
+            name: Login to DockerHub
+            uses: docker/login-action@v1 
+            with:
+                username: ${{ secrets.DOCKERHUB_USERNAME }}
+                password: ${{ secrets.DOCKERHUB_TOKEN }}
+        -
+            name: Set up QEMU
+            id: qemu
+            uses: docker/setup-qemu-action@v1
+        -
+            name: Set up Docker Buildx
+            id: buildx
+            uses: docker/setup-buildx-action@v1
+        -
+            name: Build and push
+            uses: docker/build-push-action@v2
+            with:
+                context: .
+                file: ./Dockerfile
+                platforms: linux/386,linux/amd64,linux/arm/v6,linux/arm/v7,linux/arm64,linux/ppc64le,linux/s390x
+                push: true
+                tags: |
+                    zubairov/mqtt-exporter:latest
```

Comment:
why not set it as a secret? This is not a secret but it would help to have it as a varenv